src/olympia/blocklist/management/commands/export_blocklist.py

```diff
@@ -49,50 +50,65 @@ def add_arguments(self, parser):
             default=None)
 
     def get_blocked_guids(self):
-        out = []
         blocks = Block.objects.all()
         blocks_guids = [block.guid for block in blocks]
         addons_dict = Addon.unfiltered.in_bulk(blocks_guids, field_name='guid')
         for block in blocks:
             block.addon = addons_dict.get(block.guid)
         Block.preload_addon_versions(blocks)
+        all_versions = {}
+        # First collect all the blocked versions
         for block in blocks:
             is_all_versions = (
                 block.min_version == Block.MIN and
                 block.max_version == Block.MAX)
-            versions = [
-                (block.guid, version)
-                for version in block.addon_versions.keys()
-                if is_all_versions or block.is_version_blocked(version)]
-            out.extend(versions)
-        return out
+            versions = {
+                version_id: (block.guid, version)
+                for version, (version_id, _) in block.addon_versions.items()
+                if is_all_versions or block.is_version_blocked(version)}
+            all_versions.update(versions)
+        # Now we need the cert_ids
+        cert_nums = File.objects.filter(
+            version_id__in=all_versions.keys()).values_list(
+                'version_id', 'cert_serial_num')
+        return [
+            (all_versions[version_id][0], all_versions[version_id][1], cert_nm)
+            for version_id, cert_nm in cert_nums]
 
     def get_all_guids(self):
-        return Version.unfiltered.values_list('addon__guid', 'version')
+        return File.objects.values_list(
+            'version__addon__guid', 'version__version', 'cert_serial_num')
 
     def load_json(self, json_path):
+        def tuplify(record):
+            return tuple(
+                record if len(record) == 3 else
+                record + [str(random.randint(100000, 999999))])
```

Comment:
I wouldn't do this. We can regenerate the dummy data we already have to include fake cert ids separately if needed, I don't think it should live in this command.

Reply:
fair enough - it was easier when I was developing it.

Reply:
https://github.com/mozilla/addons-server/pull/13417/commits/7b839ac67fb487e0b95aa040ae52374dc4099b16